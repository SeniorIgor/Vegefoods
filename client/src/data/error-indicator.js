export const errorIndicatorPreviewUrl = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAMAAAC6V+0/AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAABUUExURf///+fn5/Dw8HCXNCAtNPn5+qrCif7///X29uTr2Ozv5PX19bt6ZDhES9fjyHKWOcjWsZy2dbrMnHqVWVl3LPv/+anBhiw4QsrLz5yvgnGUOP/o3pAztLUAAAAJcEhZcwAAAAEAAAABAE8lxNYAAACnSURBVBjTjZCLDsMgCEURKaho23Vrs8f//+eYtt2WNMtujOjhBkEAgR+S4+sXVtUWQETDDgE/bfWIjlxNRtOWcS67V97QvNOhLGghxZggTo09SuGcc0rjGPu+WSWXQtU5GRp5q0ntIV6s6mnriMh2UWDeGKCQUIVBFLX2KYjUUfdWdQbFQDbd7MXd2K+jUV18HTzc/eXcBqWgpMph8LrCPyT2e3L860+esgYQg5UjlQAAAABJRU5ErkJggg==';
export const errorIndicatorImageUrl = '/images/not-found.svg';