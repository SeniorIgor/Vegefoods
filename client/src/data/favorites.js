export const favoritesTitleViewData = {
	title: "Избранное",
	text: "Товары для дома"
};


export const emptyFavoritesData = {
	title: "Список избранных товаров пуст!",
	text: "Нажмите на значок сердца, чтобы начать сохранять избранные товары.",
	imageUrl: '/images/empty-wishlist.svg',
	previewUrl: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAMAAAC6V+0/AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAABOUExURf///7nMndThw7q5uiIuNf///tThyNfixra2ttze3L/FutTixXCYNrW2s9fgyLe7r7vNovz++YOIiJOWl0FSNevw49HewkxRVNDjtdra2q8nv8YAAAAJcEhZcwAAAAEAAAABAE8lxNYAAACpSURBVBjTfVDbFsMgCBOLlC52ve3G///oaFc7d7azvAgxEUMIjhQOVOVWpze7tfX9p/GcQtJUk0DsWDqtDKKq6Iy10YMWECmsiRgOXdv3RIsssXdDUbak2jBbdiXKVCfhpEObflciAsYj0cztgOuLPAPyIBrHrlJmSMwwI7bbAOxfF58gfvJpRcmDLF7ebX0olvicJrfbaPC09WqmFOYLMIUfSH/br/U/ASHABr65mm1kAAAAAElFTkSuQmCC',
};