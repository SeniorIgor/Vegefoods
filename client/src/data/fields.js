
export const registerFields = [
	{
		title: "Имя",
		fieldName: "name",
		id: "register-field-name",
		type: "text",
	},
	{
		title: "Email",
		fieldName: "email",
		id: "register-field-email",
		type: "text",
	},
	{
		title: "Пароль",
		fieldName: "password",
		id: "register-field-password",
		type: "password",
	},
	{
		title: "Повторите пароль",
		fieldName: "confirm",
		id: "register-field-confirm",
		type: "password",
	},
];

export const loginFields = [
	{
		title: "Email",
		fieldName: "email",
		id: "login-field-email",
		type: "text",
	},
	{
		title: "Пароль",
		fieldName: "password",
		id: "login-field-password",
		type: "password",
	},
];

export const orderingFields = [
	{
		title: "Имя",
		fieldName: "name",
		id: "ordering-field-name",
		type: "text",
	},
	{
		title: "Email",
		fieldName: "email",
		id: "ordering-field-email",
		type: "text",
	},
	{
		title: "Телефон",
		fieldName: "phone",
		id: "ordering-field-phone",
		type: "text",
	},
	{
		title: "Адрес",
		fieldName: "address",
		id: "ordering-field-address",
		type: "text",
	}
];