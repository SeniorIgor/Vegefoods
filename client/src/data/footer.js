import React from 'react';

export const footerSocialData = [
	{
		id: 1,
		title: 'Twitter',
		image: <svg width="25" height="25" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M512 97.248c-19.04 8.352-39.328 13.888-60.48 16.576 21.76-12.992 38.368-33.408 46.176-58.016-20.288 12.096-42.688 20.64-66.56 25.408C411.872 60.704 384.416 48 354.464 48c-58.112 0-104.896 47.168-104.896 104.992 0 8.32.704 16.32 2.432 23.936-87.264-4.256-164.48-46.08-216.352-109.792-9.056 15.712-14.368 33.696-14.368 53.056 0 36.352 18.72 68.576 46.624 87.232-16.864-.32-33.408-5.216-47.424-12.928v1.152c0 51.008 36.384 93.376 84.096 103.136-8.544 2.336-17.856 3.456-27.52 3.456-6.72 0-13.504-.384-19.872-1.792 13.6 41.568 52.192 72.128 98.08 73.12-35.712 27.936-81.056 44.768-130.144 44.768-8.608 0-16.864-.384-25.12-1.44C46.496 446.88 101.6 464 161.024 464c193.152 0 298.752-160 298.752-298.688 0-4.64-.16-9.12-.384-13.568 20.832-14.784 38.336-33.248 52.608-54.496z" /></svg>,
	},
	{
		id: 2,
		title: 'Facebook',
		image: <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 470.513 470.513"><path d="M271.521 154.17v-40.541c0-6.086.28-10.8.849-14.13.567-3.335 1.857-6.615 3.859-9.853 1.999-3.236 5.236-5.47 9.706-6.708 4.476-1.24 10.424-1.858 17.85-1.858h40.539V0h-64.809c-37.5 0-64.433 8.897-80.803 26.691-16.368 17.798-24.551 44.014-24.551 78.658v48.82h-48.542v81.086h48.539v235.256h97.362V235.256h64.805l8.566-81.086h-73.37z" /></svg>,
	},
	{
		id: 3,
		title: 'Instagram',
		image: <svg width="25" height="25" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg"><path d="M261.04 512c-1.696 0-3.392 0-5.099-.008-40.132.098-77.214-.922-113.277-3.117-33.062-2.012-63.242-13.438-87.281-33.04-23.196-18.913-39.035-44.487-47.078-76.003-7-27.437-7.371-54.371-7.727-80.422C.32 300.72.055 278.57 0 256.047c.055-22.617.32-44.766.578-63.457.356-26.047.727-52.98 7.727-80.422 8.043-31.516 23.883-57.09 47.078-76.004 24.039-19.602 54.219-31.027 87.285-33.039C178.73.934 215.82-.09 256.039.008c40.145-.086 77.215.926 113.277 3.117 33.063 2.012 63.243 13.438 87.282 33.04 23.199 18.913 39.035 44.487 47.078 76.003 7 27.437 7.37 54.375 7.726 80.422.258 18.691.528 40.84.578 63.363v.094c-.05 22.523-.32 44.672-.578 63.363-.355 26.047-.722 52.98-7.726 80.422-8.043 31.516-23.88 57.09-47.078 76.004-24.04 19.602-54.22 31.027-87.282 33.039C334.781 510.977 299.305 512 261.04 512zm-5.099-40.008c39.48.094 75.73-.902 110.946-3.043 25-1.52 46.675-9.633 64.433-24.113 16.414-13.387 27.72-31.856 33.598-54.895 5.828-22.84 6.164-47.363 6.488-71.078.254-18.566.52-40.558.574-62.863-.054-22.309-.32-44.297-.574-62.863-.324-23.715-.66-48.239-6.488-71.082-5.879-23.04-17.184-41.508-33.598-54.895-17.758-14.476-39.433-22.59-64.433-24.11-35.215-2.144-71.465-3.132-110.852-3.046-39.473-.094-75.726.902-110.941 3.047-25 1.52-46.676 9.633-64.434 24.11-16.414 13.386-27.719 31.855-33.598 54.894-5.828 22.843-6.164 47.363-6.488 71.082-.254 18.582-.52 40.586-.574 62.91.055 22.226.32 44.234.574 62.816.324 23.715.66 48.239 6.488 71.078 5.88 23.04 17.184 41.508 33.598 54.895 17.758 14.476 39.434 22.59 64.434 24.11 35.215 2.144 71.476 3.144 110.847 3.046zM254.988 381c-68.922 0-125-56.074-125-125s56.078-125 125-125c68.926 0 125 56.074 125 125s-56.074 125-125 125zm0-210c-46.867 0-85 38.133-85 85s38.133 85 85 85c46.871 0 85-38.133 85-85s-38.129-85-85-85zm139-80c-16.566 0-30 13.43-30 30s13.434 30 30 30c16.57 0 30-13.43 30-30s-13.43-30-30-30zm0 0" /></svg>,
	}
];

export const footerNavMenuData = [
  {
    id: 1,
    title: "Products",
  },
  {
    id: 2,
    title: "About Us",
  },
  {
    id: 3,
    title: "Blog",
  },
  {
    id: 4,
    title: "Contacts",
  },
];

export const footerUserMenuData = [
  {
    id: 1,
    title: "Delivery",
  },
  {
    id: 2,
    title: "Exchange & Returns",
  },
  {
    id: 3,
    title: "Terms & Conditions",
  },
  {
    id: 4,
    title: "Privacy Policy",
  },
  {
    id: 5,
    title: "FAQ",
  },
];

export const footerContactsData = [
	{
		id: 1,
		title: "Moscow, Sadovnicheskaya St., 82, Bldg. 2",
		image: <svg width="20" height="25" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg"><g><path d="m407.579 87.677c-31.073-53.624-86.265-86.385-147.64-87.637-2.62-.054-5.257-.054-7.878 0-61.374 1.252-116.566 34.013-147.64 87.637-31.762 54.812-32.631 120.652-2.325 176.123l126.963 232.387c.057.103.114.206.173.308 5.586 9.709 15.593 15.505 26.77 15.505 11.176 0 21.183-5.797 26.768-15.505.059-.102.116-.205.173-.308l126.963-232.387c30.304-55.471 29.435-121.311-2.327-176.123zm-151.579 144.323c-39.701 0-72-32.299-72-72s32.299-72 72-72 72 32.299 72 72-32.298 72-72 72z" /></g></svg>,
	},
	{
		id: 2,
		title: '+7 (777) 777-77-77',
		image: <svg width="20" height="25" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 41.167 41.167"><path d="M33.506 36.816c.114.27.172.544.172.822 0 .385-.102.743-.307 1.077a1.989 1.989 0 01-.859.764l-1.73.81a9.518 9.518 0 01-2.028.669 10.163 10.163 0 01-5.351-.346c-1.086-.37-2.125-.9-3.117-1.589s-1.923-1.523-2.792-2.5a19.355 19.355 0 01-2.335-3.271 40.26 40.26 0 01-1.381-2.283c-.409-.728-.876-1.6-1.402-2.617a54.208 54.208 0 01-1.592-3.384 43.332 43.332 0 01-1.304-3.249 45.614 45.614 0 01-.745-2.31 14.977 14.977 0 01-.423-1.841 19.574 19.574 0 01-.822-5.621c0-1.203.121-2.36.362-3.473a12.322 12.322 0 011.111-3.105 10.195 10.195 0 011.899-2.568A9.331 9.331 0 0113.583.957l1.706-.785c.269-.115.539-.172.809-.172.38 0 .734.105 1.062.316.327.211.573.502.736.874l3.283 7.455a2.079 2.079 0 01-.135 1.911c-.205.337-.493.59-.865.758l-3.375 1.552a2.26 2.26 0 00-1.025.902 2.458 2.458 0 00-.356 1.282c0 .348.065.675.196.982l5.541 12.653c.217.487.516.845.899 1.074a2.34 2.34 0 002.188.135l3.375-1.552a1.93 1.93 0 011.859.144c.331.211.581.5.749.868l3.276 7.462z" fill="#000000" /></svg>,
	},
	{
		id: 3,
		title: 'email@email.com',
		image: <svg width="20" height="25" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M504.888 111.933L264.994 297.997c-2.666 2.007-5.83 3.003-8.994 3.003s-6.328-.996-8.994-3.003L7.112 111.933C2.664 118.909 0 127.13 0 136v240c0 24.814 20.186 45 45 45h422c24.814 0 45-20.186 45-45V136c0-8.87-2.664-17.091-7.112-24.067z" /><path d="M467 91H45c-4.618 0-8.987.901-13.184 2.199L256 267.25 480.184 93.199C475.987 91.901 471.618 91 467 91z" /></svg>,
	},
];
