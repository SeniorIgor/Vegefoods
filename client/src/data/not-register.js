export const notRegisterImageUrl = '/images/login.svg';
export const notRegisterPreviewUrl = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAMAAAC6V+0/AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAABXUExURf///+3s7fr6+f///eHh4fT39uTk5Pv7+/b29m+XNmKJMPPz9EFXaERZZGOHLGuUNDJETXWRRGqLP3+YWFp7UK7Bk6O4gFx4Uy1FUarCiDJFUPf/6TVETppNKugAAAAJcEhZcwAAAAEAAAABAE8lxNYAAACuSURBVBjTlc5ZDoMwDATQZLLYJuyle+9/ztqEIqT+tEYi6DG249yngvul1lT77ekqwZW8T7GzHQYKQD40dyKIAGFN1mEyIcZItCg+tqScihrQCXIYN7RYvIt2U+4OmESn4jRnN/ZGMwG9mjxtUXoNYrt6UGOIFJE7XkST05mi4YWgi9qbZ0VmeEXvtZvZMxsmPRp9eSWrlNwa5aJPqla4Xkk/th9lN70AV2d2f9Yb9kkGuY7syPgAAAAASUVORK5CYII=';