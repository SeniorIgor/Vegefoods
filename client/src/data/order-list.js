export const orderListViewData = {
	title: "Order List",
  text: "Home goods"
};

export const orderListData = {
	title: "Your order list is empty!",
  text: "Choose your favorite products from our catalog and place your first order.",
	imageUrl: '/images/empty-orderlist.svg',
	previewUrl: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAMAAAC6V+0/AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAACQUExURf////b29rLGk/r8+UNZZTNET/X19PX19f///uHi4iItN+Hh39LewW+XNE1pI/X1+Pz6+PCae/6pqF6CLcnXsJCfjO7v7rnMnyAuNj5RWFJyJDVKFx4tMnCOPf2qrq3Alfj09Yyhiy1BQT5dF+ny29Lkzu+df/KUkkphVCQuONrjxCweKcbZrLbEoJysfeXwyyzJ9wgAAAAJcEhZcwAAAAEAAAABAE8lxNYAAADiSURBVBjTXVCHcsUgDDPBxmGG7Lw9uvf//10hee21EfgQOp2wASjTmncqmNkvqFwrQIiwBqGU/se4uA3Kg/erRON9HHp4+Zc4vWsre3jb58s93ZxR2ngpj7mt7WZ3qmdRfn7ESxMCP2952B0WsYtWNucQzuZpHOGxzu+Z+KVlr0KoigIl1A8AsjWvcjz1TMRFAjkHoNqpa1tCQWmsJDrnFChvug5RK6Jqw7pwmhm4mggTUceGEXkGAO+t4Epr3VhEmySRW8qn0Hfa6kHxldUykshyslZC3XwLlMpZgtc//Yd/A1xTC+lQf0HEAAAAAElFTkSuQmCC',
};