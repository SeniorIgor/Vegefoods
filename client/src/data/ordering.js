export const orderingTitleViewData = {
	title: "Оформление заказа",
	text: "Полезные продукты"
};