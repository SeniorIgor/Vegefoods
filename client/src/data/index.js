import { loginFields, registerFields, orderingFields } from './fields';
import { cartTitleViewData, emptyCartData } from './shopping-cart';
import { favoritesTitleViewData, emptyFavoritesData } from './favorites';
import { errorIndicatorImageUrl, errorIndicatorPreviewUrl } from './error-indicator';
import { notRegisterImageUrl, notRegisterPreviewUrl } from './not-register';
import { footerContactsData, footerNavMenuData, footerSocialData, footerUserMenuData } from './footer';
import { orderListViewData, orderListData } from './order-list';
import { advantagesData } from './advantages';
import { orderingTitleViewData } from './ordering.js';

export {
	loginFields,
	registerFields,
	orderingFields,
	cartTitleViewData,
	favoritesTitleViewData,
	emptyFavoritesData,
	emptyCartData,
	errorIndicatorImageUrl,
	errorIndicatorPreviewUrl,
	notRegisterImageUrl,
	notRegisterPreviewUrl,
	footerContactsData,
	footerNavMenuData,
	footerSocialData,
	footerUserMenuData,
	orderListViewData,
	orderListData,
	advantagesData,
	orderingTitleViewData,
};