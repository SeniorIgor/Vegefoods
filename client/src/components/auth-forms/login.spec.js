import React from 'react';
import { shallow } from 'enzyme';
import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import { AuthLogin } from './index';

import Message from '../message';
import { loginFields as fields } from '../../data';
import FormField from '../form-field';

const initialProps = {
	email: '',
	password: '',
	err: null,
	errors: { email: '' },
	updateValue: () => { },
	loginUser: () => { },
};
const setUp = (props = {}) => shallow(<AuthLogin {...initialProps} {...props} />);
const setUpRender = (props = {}) => render(<AuthLogin {...initialProps} {...props} />);

describe("AUTH_LOGIN: ", () => {

	it("loginUser handler", () => {
		const loginUser = jest.fn();
		const wrapper = setUp({ loginUser });
		expect(loginUser).toHaveBeenCalledTimes(0);
		wrapper.find('.form__btn-submit').simulate('click');
		expect(loginUser).toHaveBeenCalledTimes(1);
	});

	describe('RENDER: ', () => {

		it("Message is view", () => {
			const wrapper = setUp({ err: { message: 'input not correct' } });
			expect(wrapper.find(Message)).toHaveLength(1);
		});

		it("FormFields count", () => {
			const wrapper = setUp();
			expect(wrapper.find(FormField)).toHaveLength(fields.length);
			expect(wrapper).toMatchSnapshot();
		});

		it('Name and password has props value', () => {
			const value = 'text value';
			const { getAllByDisplayValue, getByRole } = setUpRender({ email: value, password: value });
			expect(getByRole('textbox')).toHaveValue(value);

			expect(getAllByDisplayValue(value).length).toEqual(2);
		})

	})

	describe('METHODS: ', () => {

		it("change handler called updateValue", () => {
			const value = 'field';
			const fieldName = 'email';
			const updateValue = jest.fn((name, value) => `${name} ${value}`);
			setUpRender({ updateValue });
			userEvent.type(screen.getByText(/Email/i), value);
			expect(updateValue).toHaveBeenCalledTimes(value.length);
			expect(updateValue.mock.results[4].value).toEqual(`${fieldName} ${value[4]}`);
		});

	})

})