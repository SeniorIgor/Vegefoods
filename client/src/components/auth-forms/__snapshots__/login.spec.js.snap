// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AUTH_LOGIN:  RENDER:  FormFields count 1`] = `
<div
  className="auth"
>
  <h1
    className="auth__title"
  >
    Войти на сайт
  </h1>
  <div
    className="auth__form form"
  >
    <div
      className="form__field-wrap"
      key="login-field-email"
    >
      <FormField
        error=""
        fieldName="email"
        id="login-field-email"
        onChange={[Function]}
        title="Email"
        type="text"
        value=""
      />
    </div>
    <div
      className="form__field-wrap"
      key="login-field-password"
    >
      <FormField
        error=""
        fieldName="password"
        id="login-field-password"
        onChange={[Function]}
        title="Пароль"
        type="password"
        value=""
      />
    </div>
    <div
      className="form__reset-wrap"
    >
      <a
        className="form__reset-btn"
        href="#"
      >
        Забыли пароль?
      </a>
    </div>
    <button
      className="btn_order form__btn-submit"
      onClick={[Function]}
    >
      Войти
    </button>
  </div>
</div>
`;

exports[`AUTH_LOGIN: -------------------------------------------------- RENDER: ---------------------------------------------------- FormFields count 1`] = `
<div
  className="auth"
>
  <h1
    className="auth__title"
  >
    Войти на сайт
  </h1>
  <div
    className="auth__form form"
  >
    <div
      className="form__field-wrap"
      key="login-field-email"
    >
      <FormField
        error=""
        fieldName="email"
        id="login-field-email"
        onChange={[Function]}
        title="Email"
        type="text"
        value=""
      />
    </div>
    <div
      className="form__field-wrap"
      key="login-field-password"
    >
      <FormField
        error=""
        fieldName="password"
        id="login-field-password"
        onChange={[Function]}
        title="Пароль"
        type="password"
        value=""
      />
    </div>
    <div
      className="form__reset-wrap"
    >
      <a
        className="form__reset-btn"
        href="#"
      >
        Забыли пароль?
      </a>
    </div>
    <button
      className="btn_order form__btn-submit"
      onClick={[Function]}
    >
      Войти
    </button>
  </div>
</div>
`;

exports[`AUTH_LOGIN: --------------------------------------------------------- RENDER: ----------------------------------------------------------- FormFields count 1`] = `
<div
  className="auth"
>
  <h1
    className="auth__title"
  >
    Войти на сайт
  </h1>
  <div
    className="auth__form form"
  >
    <div
      className="form__field-wrap"
      key="login-field-email"
    >
      <FormField
        error=""
        fieldName="email"
        id="login-field-email"
        onChange={[Function]}
        title="Email"
        type="text"
        value=""
      />
    </div>
    <div
      className="form__field-wrap"
      key="login-field-password"
    >
      <FormField
        error=""
        fieldName="password"
        id="login-field-password"
        onChange={[Function]}
        title="Пароль"
        type="password"
        value=""
      />
    </div>
    <div
      className="form__reset-wrap"
    >
      <a
        className="form__reset-btn"
        href="#"
      >
        Забыли пароль?
      </a>
    </div>
    <button
      className="btn_order form__btn-submit"
      onClick={[Function]}
    >
      Войти
    </button>
  </div>
</div>
`;

exports[`AUTH_LOGIN: --------------------------------------------------------- RENDER: ----------------------------------------------------------- FormFields count 2`] = `
<div
  className="auth"
>
  <h1
    className="auth__title"
  >
    Войти на сайт
  </h1>
  <div
    className="auth__form form"
  >
    <div
      className="form__field-wrap"
      key="login-field-email"
    >
      <FormField
        error=""
        fieldName="email"
        id="login-field-email"
        onChange={[Function]}
        title="Email"
        type="text"
        value=""
      />
    </div>
    <div
      className="form__field-wrap"
      key="login-field-password"
    >
      <FormField
        error=""
        fieldName="password"
        id="login-field-password"
        onChange={[Function]}
        title="Пароль"
        type="password"
        value=""
      />
    </div>
    <div
      className="form__reset-wrap"
    >
      <a
        className="form__reset-btn"
        href="#"
      >
        Забыли пароль?
      </a>
    </div>
    <button
      className="btn_order form__btn-submit"
      onClick={[Function]}
    >
      Войти
    </button>
  </div>
</div>
`;
